src/handwritten/errors.py

```diff
@@ -45,18 +45,24 @@ def __init__(self, message, error_code, task_name=''):
             message (string): Specifies the error message.
             error_code (int): Specifies the NI-DAQmx error code.
         """
-        if task_name:
-            message = f'{message}\n\nTask Name: {task_name}'
-
-        super().__init__(message)
-
         self._error_code = int(error_code)
 
         try:
             self._error_type = DAQmxErrors(self._error_code)
         except ValueError:
             self._error_type = DAQmxErrors.UNKNOWN
 
+        # If message is empty, we try to put at least some information in it
+        if not message and self._error_type != DAQmxErrors.UNKNOWN:
+            message = f'{self._error_type.name} occurred. Error code: {self._error_code}.'
+        elif not message:
+            message = f'Error code: {self._error_code}.'
```

Comment:
Let's match the C API's terminology and formatting: "Status Code: -200220" (no period)

```
PS D:\dev\nidaqmx-python> poetry run python .\examples\ai_multi_task_pxie_ref_clk.py
Traceback (most recent call last):
  File "D:\dev\nidaqmx-python\examples\ai_multi_task_pxie_ref_clk.py", line 12, in <module>
    master_task.ai_channels.add_ai_voltage_chan("/PXI1Slot3/ai0")
  File "D:\dev\nidaqmx-python\generated\nidaqmx\task\collections\_ai_channel_collection.py", line 2224, in add_ai_voltage_chan
    self._interpreter.create_ai_voltage_chan(
  File "D:\dev\nidaqmx-python\generated\nidaqmx\_library_interpreter.py", line 1279, in create_ai_voltage_chan
    self.check_for_error(error_code)
  File "D:\dev\nidaqmx-python\generated\nidaqmx\_library_interpreter.py", line 6381, in check_for_error
    raise DaqError(extended_error_info, error_code)
nidaqmx.errors.DaqError: Device identifier is invalid.

Device Specified: PXI1Slot3
Suggested Device(s): Dev1

Task Name: _unnamedTask<0>

Status Code: -200220
```

Reply:
I like putting the python name, so what about:

`Status Code: -200220 (INVALID_DEVICE_ID)`

Reply:
Changed